tests/app/celery/test_tasks.py

```diff
@@ -896,13 +905,13 @@ def test_process_row_sends_save_task(
     )
     task_mock.assert_called_once_with(
         (
-            "service_id",
+            str(service.id),
             "noti_uuid",
             # signed data
             signer_mock.return_value,
         ),
         {"sender_id": str(sender_id)} if sender_id else {},
-        queue=expected_queue,
+        queue=QueueNames.DATABASE,
```

Comment:
Are all tests failing without this change?

Reply:
Missed this.. So We had to create the service and that changed the expected queue.. Hence this change.. but we don't require this.. I am redoing it at the moment.